### ISSUE TITLE
lambci cli does not find my stack

### ISSUE THREAD

**Issue by tomthorogood** [lambci/cli#4]
I have my AWS credentials and configuration in `~/.aws/credentials` and `~/.aws/config`, respectively, and can verify the existence of my stack using the AWS command line:

```
$ aws cloudformation describe-stacks --stack-name <myPrefix>-lambci
{
    "Stacks": [
        {
            "StackId": "arn:aws:cloudformation:us-west-2:<accountId>:stack/<myPrefix>-lambci/<stackId>",

# . . .
```

(Note that `<myPrefix>` is, of course, not the actual prefix.)

However, when I try using `lambci`:

```
$ lambci info --stack <myPrefix>-lambci
CLI version: 0.2.4
Stack name: <myPrefix>-lambci
Error: LambCI stack '<myPrefix>-lambci' does not exist or has no Lambda function
    at Response.<anonymous> (/usr/local/lib/node_modules/lambci-cli/actions/info.js:19:13)
    at Request.<anonymous> (/usr/local/lib/node_modules/lambci-cli/node_modules/aws-sdk/lib/request.js:364:18)
    at Request.callListeners (/usr/local/lib/node_modules/lambci-cli/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
    at Request.emit (/usr/local/lib/node_modules/lambci-cli/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
    at Request.emit (/usr/local/lib/node_modules/lambci-cli/node_modules/aws-sdk/lib/request.js:683:14)
    at Request.transition (/usr/local/lib/node_modules/lambci-cli/node_modules/aws-sdk/lib/request.js:22:10)
    at AcceptorStateMachine.runTo (/usr/local/lib/node_modules/lambci-cli/node_modules/aws-sdk/lib/state_machine.js:14:12)
    at /usr/local/lib/node_modules/lambci-cli/node_modules/aws-sdk/lib/state_machine.js:26:10
    at Request.<anonymous> (/usr/local/lib/node_modules/lambci-cli/node_modules/aws-sdk/lib/request.js:38:9)
    at Request.<anonymous> (/usr/local/lib/node_modules/lambci-cli/node_modules/aws-sdk/lib/request.js:685:12)
```

The lambci [docs](https://github.com/lambci/cli#prerequisites) only state:

> You'll need to make sure you have your AWS credentials set either in environment variables, or in configuration files, in the same format as the AWS CLI.

Is LambCI looking for these credentials in a different place than the aws CLI and other AWS-vended products (like `boto`) do? If so, where is it looking?

**Comment by mhart** [comment]
That is an excellent question. I must admit the CLI hasn't had much (any?) love over the years as the capabilities of the AWS CLI (and web console) got better. Lemme look into what it's doing – it may just need a dependency update or two.

**Comment by mhart** [comment]
I suspect it may have been an issue with picking up the region (possibly?)

I've just published a new version (v0.3.0) that may fix this – but if it doesn't, let me know how you're configuring your AWS region

**Comment by tomthorogood** [comment]
Thanks, I'll try it out. 

Currently the region is set in `~/.aws/credentials` like:

```
[default]
region = us-west-2
```

It looks like `~/.aws/config` only has an empty `[default]` line. (This configuration was generated by running `aws configure`.)

**Comment by tomthorogood** [comment]
Yep, it's definitely looking at `us-east-1` instead of the region set in the `credentials` file. 

Adding the line `region = us-west-2` to the `config` file under the `[default]` group fixed the issue!

**Comment by mhart** [comment]
Hmmm, I wonder how the AWS CLI and boto were picking it up in that case. Did you have it set in an env variable that I'm not aware of maybe?

**Comment by mhart** [comment]
Oh, wait, just re-read your message – it's set in credentials? I might need to update https://www.npmjs.com/package/awscred in that case

**Comment by mhart** [comment]
Official docs suggest it's usually stored in `~/.aws/config`: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-where

But I guess it must also check `~/.aws/credentials` if that's working for you?

**Comment by tomthorogood** [comment]
Ha ha. AWS docs are so great, because that's definitely not where its own CLI is putting it. (I just tried generating the config again to make sure it wasn't something I did manually.) Anyway, I'm OK having to add the line to `config` (if you don't think it seems like a good idea to "guess" where to find the value on the lambci side), but it might be nice to have something added to the lambci docs mentioning the discrepancy.
